lib/postprocessing/image.js

```diff
@@ -212,7 +259,9 @@ async function applyOperations(img, instructions) {
         img.resize(instructions.width || null, instructions.height || null, ">");
```

Comment:
Readability: Put `">"` in a constant and name it something like "PREVENT_UPSCALING" ?